cinema_service/settings.py

```diff
@@ -9,7 +9,7 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/4.0/ref/settings/
 """
-
+from datetime import timedelta
```

Comment:
The import statement for 'timedelta' is correct, but ensure that it is used consistently throughout the settings where time intervals are specified.